app/collectors/bucket.scala

```diff
@@ -7,72 +7,94 @@ import conf.AWS
 import controllers.routes
 import play.api.mvc.Call
 import software.amazon.awssdk.regions.Region
-import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.{GetBucketLocationRequest, ListBucketsRequest, S3Exception, Bucket => AWSBucket}
+import software.amazon.awssdk.services.s3.{S3Client, S3Configuration}
 import utils.Logging
 
 import scala.jdk.CollectionConverters._
 import scala.language.{postfixOps, reflectiveCalls}
-import scala.util.Try
 import scala.util.control.NonFatal
 
 
-class BucketCollectorSet(accounts: Accounts) extends CollectorSet[Bucket](ResourceType("bucket"), accounts, Some(Regional)) {
+class BucketCollectorSet(accounts: Accounts) extends CollectorSet[Bucket](ResourceType("bucket"), accounts, Some(Global)) {
```

Comment:
We had to create a new enum value for all of Prism's collectors when we moved to AWS SDK 2 (https://github.com/guardian/prism/pull/110). We've made the S3 Bucket Collector a Global service, so that we can crawl fewer regions (the Regional collectors crawl every region).